.github/workflows/workflows.yaml

```diff
@@ -141,9 +141,10 @@ jobs:
 
       - name: Download built zkgroup library (armhf)
         run: |
-          wget --quiet https://github.com/nanu-c/zkgroup/releases/download/v0.8.3/libzkgroup-linux-armv7-v0.8.3.so
-          install -D libzkgroup-linux-armv7-v0.8.3.so $GITHUB_WORKSPACE/lib/libzkgroup-linux-armv7.so
-          rm $GITHUB_WORKSPACE/libzkgroup-linux-armv7-v0.8.3.so
+          wget --quiet https://github.com/nanu-c/zkgroup/releases/download/v0.8.6/libzkgroup_linux_armv7_v0.8.6.so
+          mkdir $GITHUB_WORKSPACE/lib/
```

Comment:
The beauty of install, is that with the `-D` parameter, the directories are put in place for us. That means, we do not need this `mkdir` here.

I see two options here:

1) a combination of mkdir and wget. (first we make the directory, then we use wget with -O to specify the target location).

```
mkdir $GITHUB_WORKSPACE/lib/
wget --quiet https://github.com/nanu-c/zkgroup/releases/download/v0.8.6/libzkgroup_linux_armv7_v0.8.6.so -O $GITHUB_WORKSPACE/lib/libzkgroup_linux_armv7.so
```
2) a combination of wget and install (then we also need the rm in the end). Note that here no mkdir is required.

```
wget --quiet https://github.com/nanu-c/zkgroup/releases/download/v0.8.6/libzkgroup_linux_armv7_v0.8.6.so
install -D libzkgroup_linux_armv7_v0.8.6.so $GITHUB_WORKSPACE/lib/libzkgroup_linux_armv7.so
rm $GITHUB_WORKSPACE/libzkgroup_linux_armv7_v0.8.6.so
```

My vote would go for 1), as that includes fewer moving pieces, and then we do not need the rm in the end.

Reply:
Thinking about it, option 2 could be simplified by using `mv` instead of `install`: then the final rm would not be needed. Then we need the mkdir though :)

3)
```
wget --quiet https://github.com/nanu-c/zkgroup/releases/download/v0.8.6/libzkgroup_linux_armv7_v0.8.6.so
mkdir $GITHUB_WORKSPACE/lib/
mv libzkgroup_linux_armv7_v0.8.6.so $GITHUB_WORKSPACE/lib/libzkgroup_linux_armv7.so
```

Reply:
I went for option 3, thanks